docs/faq.md

```diff
@@ -0,0 +1,60 @@
+---
+description: |
+    We have colleted various questions you asked 
+    and combined them with our answers this FAQ.
+---
+
+[Contributing]: https://deno.land/manual@v1.35.1/references/contributing#submitting-a-pr-to-fresh
+[Discussions]: https://github.com/denoland/fresh/discussions
+[Roadmap]: https://github.com/denoland/fresh/issues/563
+[SaaS]: https://deno.com/saaskit
+
+<br>
+
+## Usage
+
+#### What should I **Learn** before using Fresh
+
+HTML, CSS, TypeScript and Preact
+
+<br>
+
+## Project
+
+#### Is there a **Roadmap**
+
+Currently the roadmap is tracked as a **[GitHub Issue][Roadmap]**.
+
+#### Can I talk to the **Core Maintainer**
+
+You can find **Marvin** on the following platforms:
+
+-   Discord **@marvin_h** for quick questions
+
+-   **[GitHub][Discussions]** for longer form discussions
```

Comment:
I don't think this is a good idea. Other users can help other users too. Pointing everyone in my direction means I'll be likely bombarded with support requests that others can help with too.

Reply:
How about this?
![image](https://github.com/denoland/fresh/assets/73050054/baa02c5d-7a1a-48e8-8300-6c3339c502df)